src/components/spreadsheet/table-wrapper.js

```diff
@@ -258,12 +316,18 @@ const TableWrapper = (props) => {
         }
     }, [errorMessage, snackError]);
 
+    useEffect(() => {
+        initSort(defaultSortColKey);
+        initFilters();
+    }, [tabIndex, initFilters, defaultSortColKey, initSort]);
+
     const getRows = useCallback(() => {
         if (props.disabled || !equipments) {
             return [];
         }
+        let rowData = [...equipments];
 
-        return equipments;
+        return rowData;
```

Comment:
do we need this change ?
this could trigger unnecessary rerenders : "[...equipments]"

Reply:
My bad, it's a change from the previous code when i did the filtering manually